homeassistant/components/co2signal/config_flow.py

```diff
@@ -131,16 +131,23 @@ async def async_step_reauth(
         self._reauth_entry = self.hass.config_entries.async_get_entry(
             self.context["entry_id"]
         )
+        return await self.async_step_reauth_confirm()
```

Comment:
Small change in behavior...
Previously, the original API key was tested, now the user is prompted for a new key without attempting the old key.
The tests imply that this was the desired behavior.

Reply:
Agreed, makes more sense. Thanks 👍

Reply:
Agreed, makes more sense, thanks 👍